src/components/available-organisation-units-tree/AvailableOrganisationUnitsTree.js

```diff
@@ -22,92 +23,69 @@ const defaultStyles = {
 }
 
 export class AvailableOrganisationUnitsTree extends PureComponent {
-    static propTypes = {
-        d2: PropTypes.object.isRequired,
-        onChange: PropTypes.func,
-        style: PropTypes.object,
-    }
-
-    static defaultProps = {
-        onChange: null,
-        style: defaultStyles,
-    }
-
-    constructor() {
-        super()
-
-        this.state = {
-            selected: [],
-            rootWithMember: null,
-        }
+    state = {
+        selected: [],
     }
 
     componentDidMount() {
-        if (this.state.rootWithMember == null) {
-            this.props.d2.models.organisationUnits
-                .list({
-                    paging: false,
-                    level: 1,
-                    fields:
-                        'id,displayName,path,children::isNotEmpty,memberCount',
-                })
-                .then(organisationUnitsResponse => {
-                    const organisationUnits = organisationUnitsResponse.toArray()
-                    this.setState({
-                        rootWithMembers: organisationUnits[0],
-                    })
-                })
-                .catch(() => {
-                    // TODO manage error
-                })
+        if (!this.props.organisationUnits) {
+            this.props.loadOrganisationUnits()
```

Comment:
I think loading initial state should be part of the root component for several reasons:
1. It's clear where state is being loaded when the section is initialized
    1. This makes it also easier to refactor if we move initializing state to another part of the app, e. g. redux-observalbe
1. If there's ever a scenario when we have to load multiple orgUnitTrees, this check will be done multiple times